gulpfile.js

```diff
@@ -334,6 +334,7 @@ gulp.task(
       .src(
         [
           './index.html',
+          'main.js',

```

Comment:
@jenster5  can you please replace this with the following for consistency? :
```suggestion
          './main.js',

```

Reply:
good catch done